rpc/http.go

```diff
@@ -175,6 +175,13 @@ func (hc *httpConn) doRequest(ctx context.Context, msg interface{}) (io.ReadClos
 	req.Body = ioutil.NopCloser(bytes.NewReader(body))
 	req.ContentLength = int64(len(body))
 
+	// And also add the same thing to `Request.GetBody`, which allows
```

Comment:
nit:  Consider creating a closure that would be used for both `req.Body` and `req.GetBody`.  IMO it would be easier to understand the code.

Reply:
done!